wagtail/admin/templates/wagtailadmin/generic/index_results.html

```diff
@@ -6,20 +6,25 @@
         {% include "wagtailadmin/shared/active_filters.html" with active_filters=view.active_filters %}
     {% endif %}
 
-    <div class="nice-padding">
-        {% if is_searching or is_filtering %}
+    {% if is_searching and view.show_other_searches %}
+        <div class="nice-padding">
+            {% search_other %}
+        </div>
+    {% endif %}
+
+    {% if not object_list %}
+        {# pass, to allow the same logic as `if object_list and (is_searching or is_filtering)` #}
```

Comment:
Would be nice to have something to represent `is_searching or is_filtering` in the view class, as we use this check quite extensively in the templates. Not sure what the best name is, though.

Or implementation-wise, we can also just override `is_filtering` in `IndexView` to be something like `super().is_filtering or self.is_searching`, then just use `is_filtering` in the templates (and `is_searching` for something search-specific). However, this would mean we lose the ability to detect when filters are applied and search isn't. Not sure how useful that would be, though.